anta/tests/routing/bgp.py

```diff
@@ -1404,3 +1404,141 @@ def test(self) -> None:
             self.result.is_success()
         else:
             self.result.is_failure(f"The following BGP peers are not configured or have non-zero update error counters:\n{failures}")
+
+
+class VerifyBGPPeerPrefixes(AntaTest):
+    """Verifies BGP IPv4 peer(s) consistency of prefix(s) received and accepted in a BGP session.
+
+    Expected Results
+    ----------------
+    * Success: The test will pass if the `prefixReceived` equals `prefixAccepted`, indicating that all received prefix(s) were accepted.
```

Comment:
prefixReceived and prefixAccepted seems to be json keys, please use text heading of command output in doc string and failure message to for ease of understanding

Reply:
Updated